timescaledb/tutorials/financial-candlestick-tick-data/advanced-data-management.md

```diff
@@ -0,0 +1,172 @@
+hypertables and continuous aggregates. The chunk time interval you set
+for your tick data table directly affects how these automation policies
+work. For more information, see the 
+[hypertables and chunks][chunks] section.
+
+## Hypertable chunk time intervals and automation policies
+TimescaleDB uses hypertables to provide a high-level and familiar abstraction
+layer to interact with PostgreSQL tables. You just need to access one
+hypertable to access all of your time-series data.
+
+Under the hood, TimescaleDB creates chunks based on the timestamp column.
+Each chunk size is determined by the [`chunk_time_interval`][interval]
+parameter you can provide when creating the hypertable or you can also change
+this setting afterwards. If you don't provide this optional parameter, the
+chunk time interval defaults to 7 days. This means that each of the
+chunks in the hypertable contains 7 days worth of data.
+
+It's important to understand this, because all of the TimescaleDB automation 
+policies described in this section depend on this information, and the chunk 
+time interval fundamentally affects how these policies impact your data.
+
+Let's look at these specific automation policies and how they work in the
+context of financial tick data.
+
+## Automatically delete older tick data
+Usually, the older your time-series data becomes, the less relevant and useful it is.
+This is often the case with tick data as well. As time goes on you might not
+need the raw tick data any more because you only want to query the candlestick
+aggregations. In this scenario, you can decide to remove old tick data
+automatically from your hypertable after it becomes older than a certain time
+interval.
+
+TimescaleDB has a built-in way to automatically remove raw data after a
+specific time. You can set up this automation using a
+[data retention policy][retention]:
+```sql
+SELECT add_retention_policy('crypto_ticks', INTERVAL '7 days');
+```
+
+When you run this, it adds a data retention policy to the `crypto_ticks`
+hypertable that removes a chunk after all the data in the chunk becomes
+older than 7 days. All records in the chunk need to be
+older than 7 days before the chunk is dropped. 
+
+This is where knowledge about your hypertable's chunk time interval
+becomes crucial. If you were to set a data retention policy with
+`INTERVAL '3 days'`, the policy would not remove any data after three days.
+This is because your chunk time interval is 7 days. So even after 3
+days have passed, the most recent chunk still contains data that is newer than 3
+days, and so cannot be removed by the data retention policy.
+
+If you want to change this behavior, and drop chunks more often and
+sooner, experiment with different chunk time intervals. For example, if you
+set the chunk time interval to be 2 days only, you could create a retention
+policy with a 2-day interval that would drop a chunk every other day
```

Comment:
```suggestion
policy with a two-day interval that would drop a chunk every other day
```

Reply:
Are we strict when it comes to having "two-day" vs "2-day" in general? We use the "2-day" format in a LOT of places in blogs and docs for time buckets (at least that's how I remember) and I find it easier to read (personal opinion...) @Loquacity @ryanbooz

Reply:
I find the number easier to read as well, so I'm totally happy to change the style guide if that's the consensus

(enforcing style guides means defending opinions you disagree with all the time 😅 )

Reply:
Given our discussion in Slack (specifically here: https://iobeam.slack.com/archives/C0J94TE4F/p1646945388271069), @zseta can ignore these suggestions and keep the intervals at #-(day/minute/hour).

We'll update our style-guide accordingly.